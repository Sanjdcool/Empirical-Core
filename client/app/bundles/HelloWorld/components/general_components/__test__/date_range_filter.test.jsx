import React from 'react';
import { shallow } from 'enzyme';

import DateRangeFilter from '../date_range_filter'

describe('DateRangeFilter component', () => {
  it('should render', () => {
    const wrapper = shallow(
      <DateRangeFilter selectDates={() => {}} />
    );
    expect(wrapper).toMatchSnapshot();
  })
})
