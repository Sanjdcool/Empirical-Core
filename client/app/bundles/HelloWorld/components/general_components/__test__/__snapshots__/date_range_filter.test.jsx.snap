// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DateRangeFilter component should render 1`] = `
<div
  className="row date-range-filter"
>
  <div
    className="no-pl col-xs-6 col-sm-5"
  >
    <DatePicker
      dateFormatCalendar="MMMM YYYY"
      disabled={false}
      disabledKeyboardNavigation={false}
      dropdownMode="scroll"
      maxDate={"2017-04-05T20:06:02.845Z"}
      monthsShown={1}
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      onMonthChange={[Function]}
      placeholderText="Completed: From"
      popoverAttachment="top left"
      popoverTargetAttachment="bottom left"
      popoverTargetOffset="10px 0"
      selected={null}
      tetherConstraints={
        Array [
          Object {
            "attachment": "together",
            "to": "window",
          },
        ]
      }
      utcOffset={-240}
      withPortal={false}
    />
  </div>
  <div
    className="no-pl col-xs-6 col-sm-5"
  >
    <DatePicker
      dateFormatCalendar="MMMM YYYY"
      disabled={false}
      disabledKeyboardNavigation={false}
      dropdownMode="scroll"
      maxDate={"2017-04-05T20:06:02.845Z"}
      monthsShown={1}
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      onMonthChange={[Function]}
      placeholderText="Completed: To"
      popoverAttachment="top left"
      popoverTargetAttachment="bottom left"
      popoverTargetOffset="10px 0"
      selected={null}
      tetherConstraints={
        Array [
          Object {
            "attachment": "together",
            "to": "window",
          },
        ]
      }
      utcOffset={-240}
      withPortal={false}
    />
  </div>
</div>
`;
